pages/_document.js

```diff
@@ -24,7 +25,7 @@ class MyDocument extends Document {
       <Html>
         <Head>
           <link rel="preconnect" href="https://fonts.googleapis.com" />
-          <link rel="preconnect" href="https://fonts.gstatic.com" crossOrigin />
+          <link rel="preconnect" href="https://fonts.gstatic.com" crossOrigin={"true"} />
```

Comment:
was seeing:

`Warning: Received `true` for a non-boolean attribute `crossOrigin`.`

locally. So setting it to non boolean since this is a valid HTML attribute